docs/references/java_project/index.md

```diff
@@ -1,17 +1,96 @@
+
+This directory contains the package structure specified during `mp-init`. Inside the package, every template creates an
+Adapter.java file that serves as the main class for the project. 
+
+### build.gradle.kts (file)
+
+This files contains all the project dependencies along with some building specifications used by the Dockerfile. 
+
+### commands.cfg (file)
+This file contains a list of the commands the HTTP server can run, along with the path to the executable related to the
+command. By default, all commands are run by executing the `adapter.py` file along with a parameter that defines a command.
+For example, when the HTTP server receives a request to run a test connection, it reads the commands.cfg key for `test`
+and runs the process defined by the key value, `/usr/local/bin/python app/adapter.py test`.
+
+---
+#### Sample Adapter:
+
+The template adapter collects several objects and metrics from the JVM that the adapter is running in,
+and can be used as a starting point for creating a new adapter. 
+The template adapter uses Java Integration SDK library to streamline the process of building adapter.
+The template adapter has comments throughout the code to help new users understand the process of creating their own
+adapter using the existing code. 
+For additional guidance creating adapters see our `Guides` section.
+
+#### New Adapter:
+
+The new adapter comes with the minimum required code to develop your own adapter.
+
 ### Adapter methods (required)
 
-The methods in the adapter source code are required, and should be modified to generate a custom
-adapter. Each method fulfills a request from the VMware Aria Operations collector, and can be tested individually using
-`mp-test` (covered in [Testing a Management Pack](#testing-a-management-pack)).
+The methods in the Adapter.java source code are required, and should be modified to generate a custom adapter.
+Each method fulfills a request from the
+VMware Aria Operations collector, and can be tested individually
+using`mp-test` (covered in [Testing a Management Pack](../../get_started.md#testing-a-management-pack)).
 
 - `test(adapterInstance)`:
   Performs a test connection using the information given to the adapterInstance to verify the adapter instance has been configured properly.
   A typical test connection will generally consist of:
 
-    1. Read identifier values from adapterInstance that are required to connect to the target(s)
-    2. Connect to the target(s), and retrieve some sample data
-    3. If any of the above failed, return an error, otherwise pass.
-    4. Disconnect cleanly from the target (ensure this happens even if an error occurs)
+  1. Read identifier values from adapterInstance that are required to connect to the target(s)
+  2. Connect to the target(s), and retrieve some sample data
+  3. If any of the above failed, return an error, otherwise pass.
+  4. Disconnect cleanly from the target (ensure this happens even if an error occurs)
 
```

Comment:
For some reason these nested lists aren't rendering correctly:
<img width="782" alt="image" src="https://github.com/vmware/vmware-aria-operations-integration-sdk/assets/3310870/5291cac4-f0fe-4475-b14b-d64627b6970d">

Reply:
These might work well as definition lists as well -- see comment here: https://github.com/vmware/vmware-aria-operations-integration-sdk/pull/312#discussion_r1355355598